code/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Fjalla+One&display=swap" rel="stylesheet">
+  <title>Malin's weather app</title>
+</head>
+
+<body id="body">
+
+  <section class="todays-details">
+    <p> <span id= "weatherConditions"> </span> | <span id= "temperature"> </span></p>    
+  </section>
+
+  <section class="sun-up-down">
+    <p id = "sunrise">Sunrise </p>
+    <p id = "sunset">Sunset </p>
+  </section>
+
+  <section class="weather-image-section">
+    <img id="weatherConditionsImage" src="" alt="img todays weather">
```

Comment:
You actually don't need to add "img" in the alt text
```suggestion
    <img id="weatherConditionsImage" src="" alt="todays weather">
```